mava/networks.py

```diff
@@ -98,6 +99,23 @@ def __call__(self, observation: Union[Observation, ObservationGlobalState]) -> c
         return jnp.squeeze(critic_output, axis=-1)
 
 
+class ContinuousFFActor(nn.Module):
```

Comment:
Might be nice to have a log_std that's just a network param and see how that works? It's how a lot of [other PPO implementations](https://github.com/FLAIROx/JaxMARL/blob/main/baselines/IPPO/ippo_ff_mabrax.py#L42) do it. Could you test this and add it as a separate type of network if it works well?